pkg/json/encode.go

```diff
@@ -40,3 +40,27 @@ func (s Set) MarshalJSON() ([]byte, error) {
 	buf.WriteString(`]]`)
 	return buf.Bytes(), nil
 }
+
+func (cmr *CondMonitorParameters) UnmarshalJSON(p []byte) error {
```

Comment:
how is this related to the klog change ? better to separate as it makes reviewing and merging 10x  faster

Reply:
It's from the previous PR, this PR is based on the first one